### PR TITLE
update translations

### PR DESCRIPTION
**Pull request by g1039**
@miltontony @hodgestar Please review

### CODE REVIEW

**Comment by g1039** [comment]
@miltontony Ready for review

File: development.ini [hunk 1]
```diff
@@ -24,6 +24,10 @@ available_languages =
     ara_AE
     rus_RU
     kir_KG
+    bur_MM
+    pt_BR
```
Comment (hodgestar) [inline development.ini]: We're using three letter language codes for the other languages. Just checking that its okay to be using a two-letter code here?

**Comment by hodgestar** [comment]
Left one question, otherwise looks good to me.

File: development.ini (hunk 1 above)
Comment (miltontony) [inline development.ini]: no, lets use 3 letters everywhere

**Comment by hodgestar** [comment]
I've also suggested that we load `kur_TR` as `ckb_IQ` as the nearest equivalent that Babel supports (see http://www-01.sil.org/iso639-3/documentation.asp?id=kur -- `ckb_IQ` is the closest alternative that current Babel supports).

**Comment by g1039** [comment]
@miltontony @hodgestar READY FOR REVIEW

**Comment by hodgestar** [comment]
:+1:

File: springboard_iogt/locale/ara_AE/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -33,10 +34,10 @@ msgstr ""
 
 #: springboard_iogt/utils.py:26
 msgid "Your Rights"
-msgstr ""
+msgstr "حقوقك"
 
 #: springboard_iogt/utils.py:27
-msgid "Free legal information & support"
+msgid "Know and use them!"
```
Comment (miltontony) [inline springboard_iogt/locale/ara_AE/LC_MESSAGES/messages.po:40]: @TrevorOctober I have concern about this change - we'll need to reupload the english POT file and have Botswang update "Know and use them!" for all the translations.

Comment (miltontony) [inline springboard_iogt/locale/ara_AE/LC_MESSAGES/messages.po:40]: This PR can still land though, but we'll have to do another PR with the new translations for this sentence
